### ISSUE TITLE
Standard format

### ISSUE THREAD

**Issue by rEnr3n** [feediron/feediron-recipes#1]
I noticed some recipe file names end with tld (e.g. arstechnica.com) and some file names do not have one (e.g. zdnet).

Can we make this a standard for file names?
`domain.tld`

As for the case of different recipes on the same domain (e.g. heise.de)
Maybe restructure the config like this?
```
    "config": {
        "match": "heise.de\/tr",
        ...
        },
              {
        "match": "heise.de\/tp",
        ...
        }
```

**Comment by dugite-code** [comment]
Yes I actually started re-naming some of the recipes yesterday, but I didn't finish.

I defiantly agree a standard naming convention is needed. I also plan on adding category folders with a selection dialogue in the plugin. 

I had initially gone with a more descriptive `heise.de-developer` but now I can see this will cause issues with localization (long off lofty goal for now) so perhaps your suggestion of combining the different sub-folder recipes into their respective domain has merit

**Comment by uqs** [comment]
It would be good to have them end in .json so that editors can do the proper indentation and syntax highlighting. Also, it might allow for different formats of the recipes, because JSON doesn't allow for comments and these recipes desperately need to be commented to help future maintenance.

**Comment by dugite-code** [comment]
I've changed all the files to .json format and added a change to the Recipe manager to remove the .json from the file-name for ease of reading.

As for comments, until I have a chance to really re-work the storage back-end stuff any key value pare can be used in the config for comment saving.

```json
"example.tld": {
        "type": "readability",
        "comments": "Add a filter rule in TT-RSS to drop pages with tag 'bad article'",
        "prependimage": true,
        "appendimages": true
    }
```

I've also settled on a `domain.tld-subdir` naming convention for now.
